src/frontend/apps/desk/src/features/teams/components/ModalUpdateTeam.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  Button,
+  Input,
+  Loader,
+  Modal,
+  ModalSize,
+  VariantType,
+  useToastProvider,
+} from '@openfun/cunningham-react';
+import { t } from 'i18next';
+import { useState } from 'react';
+
+import { Box, Text, TextErrors } from '@/components';
+import useCunninghamTheme from '@/cunningham/useCunninghamTheme';
+
+import { Team, useUpdateTeam } from '../api';
+import IconEdit from '../assets/icon-edit.svg';
+
+interface ModalUpdateTeamProps {
+  onClose: () => void;
+  team: Team;
+}
+
+export const ModalUpdateTeam = ({ onClose, team }: ModalUpdateTeamProps) => {
```

Comment:
The app is not responsive yet, but the modal seems to cause some layout issues

<img width="392" alt="Capture d’écran 2024-03-21 à 15 18 30" src="https://github.com/numerique-gouv/people/assets/45729124/b01ddeca-9ea4-4630-b461-755bc628b16b">